cross/zerotier/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = ZeroTierOne
+PKG_VERS = 1.2.12
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/zerotier/ZeroTierOne/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = https://www.zerotier.com
+COMMENT  = A Simple global P2P encrypted LAN for all of your devices.
+LICENSE  = GPLv3
+
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = HAVE_CXX=yes
+CONFIGURE_TARGET = myConfigure
```

Comment:
To ease debug you should name Makefile targets "zerotier_custom_configure" and "zerotier_custom_install"